templates/template_data.go

```diff
@@ -24,7 +25,14 @@ type Template = template.Template
 type KV = template.KV
 type Data = template.Data
 
-var FromGlobs = template.FromGlobs
+var New = template.New
```

Comment:
I haven't looked - but want to make sure I ask the question, do we need this export?